cnv/cnv_users_guide/cnv-enabling-user-permissions-to-clone-datavolumes.adoc

```diff
@@ -0,0 +1,24 @@
+[id="cnv-enabling-user-permissions-to-clone-datavolumes"]
+= Enabling user permissions to clone DataVolumes across namespaces
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-enabling-user-permissions-to-clone-datavolumes
+toc::[]
+
+The isolating nature of namespaces means that users cannot by default 
+clone resources between namespaces. 
+
+To enable a user to clone a virtual machine to another namespace, a 
+user with the ClusterAdmin role must create a new ClusterRole. Bind 
```

Comment:
I haven't seen "ClusterAdmin" in our docs -- only `cluster-admin`. Can you confirm that "ClusterAdmin" is right?